### ISSUE TITLE
Is there a way to get real-time SizeInfo when sliding the BottomSheet?

### ISSUE THREAD

**Issue by Richard6536** [lodev09/react-native-true-sheet#84]
Hi,

I’m using TrueSheet in my React Native project and I’m trying to get real-time updates on the SizeInfo (similar to what onSizeChange provides) while I’m sliding the BottomSheet. Currently, onSizeChange only triggers once the sliding gesture is completed. Is there any existing method or workaround that would allow me to get the SizeInfo continuously during the slide?

Thanks!

**Comment by lodev09** [comment]
Yep.. It will come soon, just didn't get the chance to work on it yet -- should also support reanimated :)

**Comment by Richard6536** [comment]
> Yep.. It will come soon, just didn't get the chance to work on it yet -- should also support reanimated :)

Thank you! I'm looking forward to it. Appreciate your hard work on this!

**Comment by erickreutz** [comment]
This would be really helpful to have!

**Comment by lodev09** [comment]
@erickreutz thanks. I was about to implement this but lost track. reopening

**Comment by lodev09** [comment]
I'm going to take a look at some of new arch bugs first then will do this next. Thanks for the sponsor @erickreutz !

**Comment by erickreutz** [comment]
> I'm going to take a look at some of new arch bugs first then will do this next. Thanks for the sponsor [@erickreutz](https://github.com/erickreutz) !

@lodev09 sounds good thank you!
